### DISCUSSION TITLE
VueDX warning for custom elements

### DISCUSSION THREAD

**Discussion by Duncank** [vuedx/languagetools#224] [Q&A] [answered]
I'm using a few custom elements in my app, like `<icon>`, and VueDX reports this warning for that:
> Cannot find a global component matching: 'icon'. VueDX(59001).

Since this is not a component I cannot include it in a list of global components, so I'm wondering how to solve this. 
For Vue itself, I've included it in the compilerOptions as follows:

```js
compilerOptions: {
  isCustomElement: (tag) => ['icon'].includes(tag),
}
```

**Comment by znck** [comment, marked as answer]
Use [`JSX.IntrinsicElements`](https://www.typescriptlang.org/docs/handbook/jsx.html#intrinsic-elements) to define custom elements.
